conduct/COC_PROCESS_FOR_INCIDENT_MANAGEMENT.md

```diff
@@ -51,7 +53,7 @@ All members of the CPC are subscribed to the `report@openjsf.org` mailing list.
 
 Note: We understand that the reporter and the target can be separate persons. In this case the CPC will communicate with the reporter unless the target gives permission for the CPC to communicate with them.
 
-All members of the [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md#code-of-conduct-panel)
+All members of the [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/COC_POLICY.md#code-of-conduct-panel)
```

Comment:
Should there references here and below to the CoCP be updated?

Reply:
Thanks for catching that!